values-cs/full_description.txt

```diff
@@ -1,17 +1,17 @@
 Full description (4000 character limit)
 ======================================
 
-<b>2048++</b> is hands down the cleanest, most responsive 2048 app for Android.
+2048++ je nečistší a nejlepší 2048 aplikace pro android. 
 
-* 3 free game modes/variants: <b>2048, 243, and 16384.</b>
-* 4 additional game modes/variants available as in-app purchases: <b>Hex, Blitz, Fibonacci, and 144.</b>
-* A simple design with a colorful, Material Design inspired user interface.
-* Insanely smooth and realistic animations.
-* Synchronize your high scores across all your devices.
-* Sign in with Google Play Games for leaderboards and achievements.
-* 100% native and 100% Ad-Free.
+* 3 herní režimy zdarma: 2048, 243 a 16384.
+* 4 další herní režimy dostupné za in-app nákupy: hex, blitz, fibonacci a 144.
```

Comment:
Is it typical to capitalize names in Czech? (i.e. should "hex, blitz, fibonacci" be "Hex, Blitz, Fibonacci"?)